draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -128,24 +139,48 @@ informative:
 
 --- abstract
 
-This document defines generic techniques to achive hybrid
-post-quantum/traditional (PQ/T) key encapsulation mechanisms (KEMs) from
-post-quantum and traditional component algorithms that meet specified
-security properties.
+"Post-quantum" (PQ) algorithms promise to resist attack by a quantum computer,
```

Comment:
Feel free to ignore this stylistic blabbering: personally I like to write "post-quantum algorithms are designed to resist attack of quantum computers". Then we don't need any qualifications like "promise" or "believe to" etc. Of course it's not completely true: several schemes were first designed before quantum computers were known about.